cypress/e2e/gui/customers.cy.js

```diff
@@ -0,0 +1,299 @@
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_URL = `${API_URL}/customers`;
+
+describe('Customers', () => {
+  beforeEach(() => {
+    cy.setCookie('cookieConsent', 'accepted');
+    cy.visit('/');
+  });
+
+  context('View', () => {
+    it('goes back to the customers list when clicking the "Back" button', () => {
+      cy.contains('button', 'View').click();
+      cy.contains('button', 'Back').click();
+
+      cy.contains('p', 'Below is our customer list.').should('be.visible');
+    });
+
+    it('shows a Loading... fallback element before the initial customers fetch', () => {
+      cy.intercept(
+        'GET',
+        `${CUSTOMERS_URL}?page=1&limit=10&size=All&industry=All`,
+        (req) => {
+          req.reply((res) => {
+            res.setDelay(4000);
```

Comment:
```suggestion
            res.setDelay(1000);
```

Um _delay_ de `1000` já é suficiente para o Cypress verificar o estado transitório.

Lembre-se: testes devem fornecer _feeback_ rápida aos times.